src/components/elements/Map-mapbox/GeoJSON.d.ts

```diff
@@ -0,0 +1,64 @@
+export as namespace GeoJSON;
+export type GeoJsonGeometryTypes = Geometry["type"];
+
+export type GeoJsonTypes = GeoJSON["type"];
+
+export type BBox = [number, number, number, number];
+
+export type Position = number[];
+
+export interface GeoJsonObject {
+  type: GeoJsonTypes;
+  bbox?: BBox | undefined;
+}
+export type GeoJSON = Geometry | Feature | FeatureCollection;
+export type Geometry = Point | MultiPoint | LineString | MultiLineString | Polygon | MultiPolygon | GeometryCollection;
+export type GeometryObject = Geometry;
+
+export interface Point extends GeoJsonObject {
+  type: "Point";
+  coordinates: Position;
+}
+
+export interface MultiPoint extends GeoJsonObject {
+  type: "MultiPoint";
+  coordinates: Position[];
+}
+
+export interface LineString extends GeoJsonObject {
+  type: "LineString";
+  coordinates: Position[];
+}
+
+export interface MultiLineString extends GeoJsonObject {
+  type: "MultiLineString";
+  coordinates: Position[][];
+}
+export interface Polygon extends GeoJsonObject {
+  type: "Polygon";
+  coordinates: Position[][];
+}
+
+export interface MultiPolygon extends GeoJsonObject {
+  type: "MultiPolygon";
+  coordinates: Position[][][];
+}
+// uncomment this block if you need the GeometryCollection type
+// export interface GeometryCollection<G extends Geometry = Geometry> extends GeoJsonObject {
+//     type: "GeometryCollection";
+//     geometries: G[];
+// }
```

Comment:
If we're not going to use this, please remove from the codebase.

Reply:
Removed.